packages/server/src/service/pdfForm/util.ts

```diff
@@ -30,6 +30,23 @@ export const splitFullName = (fullName: string) => {
   return [firstName, middleName, lastName, suffix]
 }
 
+export const stripPhoneNumber = (phoneNumber: string) => {
+  // Get rid of characters. split-join acts as 'replaceAll'
+  let strippedNumber = phoneNumber
+    .replace('+', '')
+    .replace('(', '')
+    .replace(')', '')
+    .split('-')
+    .join('')
+    .split(' ')
```

Comment:
`replace(' ', '')` and get rid of `.split(' ')`

Reply:
Same as with '-' : replaceAll